lib/pengine/utils.c

```diff
@@ -148,12 +148,12 @@ node_list_exclude(GHashTable * hash, GList *list, gboolean merge_scores)
  * \return Hash table equivalent of node list
  */
 GHashTable *
-pe__node_list2table(GList *list)
+pe__node_list2table(const GList *list)
 {
     GHashTable *result = NULL;
 
     result = pcmk__strkey_table(NULL, free);
-    for (GList *gIter = list; gIter != NULL; gIter = gIter->next) {
+    for (const GList *gIter = list; gIter != NULL; gIter = gIter->next) {
         pe_node_t *new_node = pe__copy_node((pe_node_t *) gIter->data);
```

Comment:
technically we should probably cast `gIter->data` to `const pe_node_t *`, but it makes absolutely no difference